python/lsst/obs/auxTel/auxTelMapper.py

```diff
@@ -0,0 +1,338 @@
+                for c in channels:
+                    did = list(value)
+                    did.insert(channelIndex, c)
+                    dids.append(tuple(did))
+
+        return dids
+    #
+    # The composite type "raw" doesn't provide e.g. query_raw, so we defined type _raw in the .paf file
+    # with the same template, and forward requests as necessary
+    #
+
+    def query_raw(self, *args, **kwargs):
+        """Magic method that is called automatically if it exists.
+
+        This code redirects the call to the right place, necessary because of leading underscore on _raw.
+        """
+        return self.query__raw(*args, **kwargs)
+
+    def map_raw_md(self, *args, **kwargs):
+        """Magic method that is called automatically if it exists.
+
+        This code redirects the call to the right place, necessary because of leading underscore on _raw.
+        """
+        return self.map__raw_md(*args, **kwargs)
+
+    def map_raw_filename(self, *args, **kwargs):
+        """Magic method that is called automatically if it exists.
+
+        This code redirects the call to the right place, necessary because of leading underscore on _raw.
+        """
+        return self.map__raw_filename(*args, **kwargs)
+
+    def bypass_raw_filename(self, *args, **kwargs):
+        """Magic method that is called automatically if it exists.
+
+        This code redirects the call to the right place, necessary because of leading underscore on _raw.
+        """
+        return self.bypass__raw_filename(*args, **kwargs)
+
+    def map_raw_visitInfo(self, *args, **kwargs):
+        """Magic method that is called automatically if it exists.
+
+        This code redirects the call to the right place, necessary because of leading underscore on _raw.
+        """
+        return self.map__raw_visitInfo(*args, **kwargs)
+
+    def bypass_raw_visitInfo(self, datasetType, pythonType, location, dataId):
+        if False:
+            # afwImage.readMetadata() doesn't honour [hdu] suffixes in filenames
+            #
+            # We could workaround this by moving the "else" block into obs_base,
+            # or by changing afw
+            #
+            return self.bypass__raw_visitInfo(datasetType, pythonType, location, dataId)
+        else:
+            import re
+            import lsst.afw.image as afwImage
+
+            fileName = location.getLocationsWithRoot()[0]
+            mat = re.search(r"\[(\d+)\]$", fileName)
+            if mat:
+                hdu = int(mat.group(1))
+                md = afwImage.readMetadata(fileName, hdu=hdu)
+            else:
+                md = afwImage.readMetadata(fileName)  # or hdu = INT_MIN; -(1 << 31)
+
+            return afwImage.VisitInfo(md)
+
+    def std_raw_amp(self, item, dataId):
+        return self._standardizeExposure(self.exposures['raw_amp'], item, dataId,
+                                         trimmed=False, setVisitInfo=False)
+
+    def X_validate(self, dataId):
```

Comment:
I don't understand all these "not currently live" methods.  Is it that you are planning on using these in the future but do not at the moment?  If that's the case, I think we can remove them here and leave them on a branch or something.

Reply:
I believe this was an attempt, during the birth of the previous package to mitigate the snowballing cargo-culting, and by "turning them off", and seeing which were actually necessary when it came to runtime. Given that this is the second package which has inherited them with them being re-enabled (albeit without processing sky-data) I've just gone and deleted them for now. We can always re-cargo-cult them from other packages, so I don't think there is a need for a branch here (plus there is always the git history)